docs/static_site/src/_includes/head.html

```diff
@@ -15,4 +15,25 @@
 
   <script src="{{'/assets/js/clipboard.js'|relative_url}}"></script>
   <script src="{{'/assets/js/copycode.js'|relative_url}}"></script>
+  <style>
```

Comment:
Duplicated in layout.html. Should be abstracted to a single source so it's easier to manage.